packages/components/addon/components/hds/app-footer/index.hbs

```diff
@@ -0,0 +1,24 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<div class="hds-app-footer" ...attributes>
+  {{yield (hash ExtraContentBefore=(component "hds/app-footer/extra-content-before"))}}
+
+  <ul class="hds-app-footer__list" aria-label="Footer items">
```

Comment:
I basically just put something (for the aria-label) so that it's there but didn't know the best value. Would like @MelSumner's feedback on this.

Reply:
I think this aria-label value is fine for now. We might want to eventually make it so it will support i18n but as far as I know, teams have some i18n implemented but don't actually support multiple languages. 

Out of curiosity, what happens when you run the linter on this file? Does it say anything about no bare strings? If so, I'd add an inline disable for that rule, so it doesn't cause issues for our consumers (they have to pass lint rules to merge).

Reply:
Thanks! Is there an extra linter to run? I run the normal linter and fix issues before I push code up. (And if I miss something then the automated checks don't pass and I fix then.)

I can add arguments to expose the aria-label text.

Reply:
I added getters to expose the aria-label values now so marking this as resolved.